action.yml

```diff
@@ -86,6 +95,33 @@ runs:
         VERSION=$(git tag --list 'firtool-*' | tail -n1)
         echo version=$VERSION >> $GITHUB_OUTPUT
         echo Latest version is: \`$VERSION\` >> $GITHUB_STEP_SUMMARY
+    - name: Decide whether or not to open PR
+      id: open-pr
+      shell: bash
+      run: |
+        VERSION=${{ steps.get-version-latest.outputs.version }}
+        if [[ ${{ steps.get-version-current.outputs.version }} != "$VERSION" ]]; then
+          if [[ -n "${{ inputs.should-create-pr }}" ]]; then
+            if $(downstream/${{ inputs.should-create-pr }} "$VERSION"); then
+              RESULT="0"
+            else
+              RESULT=$?
+            fi
+          else
+            RESULT="0"
+          fi
+          if [[ "$RESULT" -eq "0" ]]; then
+            echo "result=yes" >> $GITHUB_OUTPUT
+          elif [[ "$RESULT" -eq "1" ]]; then
+            echo "::notice::User provided ${{ inputs.should-create-pr }} returned 1, skipping PR creation..."
```

Comment:
What does `::notice::User` do?

Reply:
Documented-ish here: https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/workflow-commands-for-github-actions#setting-a-notice-message

Here's what it looks like in the Summary page:
<img width="1264" alt="Screenshot 2024-08-06 at 8 36 26 AM" src="https://github.com/user-attachments/assets/641f78fd-c63c-4711-b85b-8bfe2179a1d4">

And in the log of the job:
<img width="924" alt="Screenshot 2024-08-06 at 8 37 13 AM" src="https://github.com/user-attachments/assets/9d591af4-b36f-461c-b0ed-4256aefed4b7">

Kinda nice IMO!

Reply:
Ah, cool. 👍